cmd/crdvalidator/README.md

```diff
@@ -0,0 +1,116 @@
+# crdvalidator
+
+## Summary
+A part of the core value proposition for RukPak is the safe upgrade of `CRD`s. As a result, this repository comes equipped with a "CRD Validator Webhook", `crdvalidator`, that will validate specified `CRD` upgrade events.
```

Comment:
nit: could we use the full name of the resource and then use the abbreviation from then on? So CustomResourceDefinition (CRD). Also maybe a link to the upstream k8s docs on CustomResourceDefinitions could be helpful here as well?